### ISSUE TITLE
WCAG 2.4.1 ✅ (Supports): Bypass Blocks

### ISSUE THREAD

**Issue by StephDriver** [BirkbeckCTP/janeway#5101]
## WCAG 2.4.1: Bypass Blocks

**Level:** A
**Principle:** Operable
**Result:** ✅ Supports (all themes)

### Description
This issue tracks the audit and implementation of WCAG 2.2 Success Criterion 2.4.1: Bypass Blocks.

### Resources
- [WCAG 2.2 Success Criterion 2.4.1](https://www.w3.org/WAI/WCAG22/Understanding/bypass-blocks.html)

### Audit Checklist
- [x] Review previous Audit Result
- [x] Review work identified in previous audit and list open issues.
- [x] Check for any other relevant open issues.
- [x] Retest open issues, are these issues still present?
- [x] Test for any new issues that may have been introduced.
- [x] Determine compliance status, add this to issue name.
- [x] Add any new issues to the Accessibility project for triage.
- [x] Replace previous audit in milestone with this update.

### Previous audit
> ❌ Bypass blocks
Bypass blocks are not in use and should be.

[#4131]

### Progress since last audit

The last audit identified the following work:
- #3932
- #4225
- #4242

And these issues have also been completed which include bypass work:
- #4337
- #3242

"Skip to main" links have been added as a common template for all themes such that this is on all front of house pages (Technique G1).  Work has been done to improve the structure of pages with landmarks (Techniques ARIA11 and H97) and contextual headings (Technique H69).  We reviewed our use of lists and converted a substantial amount of related content into lists to provide better context and navigation within the list.

 ### Audit Notes
The goal here is to "Provide a means of skipping repeating content."  For Janeway, repeating content primarily refers to our header, footer and navigation. We do not have advertising frames.   We have a skip to main link at the top of every page, that only appears on keyboard navigation, and which bypasses our header and nav.   The footer is the last content on the page, so no bypass required.  

We have particularly looked at the article page in terms of how to users may read an article, and need to pass back and forth between parts of the page while reading, for example having checked an endnote or reference, to go back to their original place in the text.   The updates include not just links back, but a brief highlighting of the landing location in order to help readers reorientate.   In this case it is not the content itself which is repeating, but rather than a user may be reasonably expected to move back and forth bypassing the same content multiple times as they consume that content. 

### Automated Testing
No violations were found for any of the following Axe rules:
- bypass — Page must have means to bypass repeated blocks
- skip-link — The skip-link target should exist and be focusable

 ### Work arising

As our experience with the article page has shown, the nature of Janeway is that we have content that may only appear once on a page, but may need to be viewed or found repeatedly during a user's visit to that page.  While not within the letter of the Bypass Block success criterion, this anticipated user journey does fit the spirit of 'skipping repeating content'. 

We are compliant with this success criterion, but it is likely that additional bypass blocks over content that only appears once but a user will skip over multiple times, could improve our accessibility and usability.  The best way to identify these locations is from user input, as they know what they are wanting to skip. Back to text links were a user-request. We need to keep open to further user requests for skip links or bypass blocks.
